scripts/publish.sh

```diff
@@ -259,7 +308,7 @@ match_two_sha() {
 
 if [ "${1}" = "publish" ]; then
 	if [ "${2}" = "dockerhub" ]; then
-		publish_to_docker_hub amazon/aws-for-fluent-bit:latest amazon/aws-for-fluent-bit:${AWS_FOR_FLUENT_BIT_VERSION}
+		publish_to_docker_hub amazon/aws-for-fluent-bit meghnaprabhu/aws-for-fluent-bit
```

Comment:
Need to replace with `amazon`.

Reply:
Yes, I will replace that as the last step along with the account ID

Reply:
Also, I can see both of the values are same. If we are not using the 2nd parameter (fluent-bit-version), we can remove it.